graphql-schemas/schema.yml

```diff
@@ -756,6 +757,24 @@
   - { name: user, type: User_v1, isRequired: true }
   - { name: requestId, type: string, isRequired: true }
 
+- name: AWSAccountSharingOption_v1
+  isInterface: true
```

Comment:
what is the idea here? e.g. also share stuff like RDS snapshots etc?

Reply:
perhaps. it's a future compatible design.